InstallingDrivers.md

```diff
@@ -120,3 +120,78 @@ _Note: Only Skylake, Kaby Lake, and Coffee Lake offer full Vulkan support. Broad
 To install support for Vulkan API (will be functional only if you have a [Vulkan capable GPU](https://en.wikipedia.org/wiki/Vulkan_(API)#Compatibility) and driver), execute following command:
 
     sudo dnf install vulkan-loader vulkan-loader.i686
+
+    
+## openSUSE Tumbleweed/Leap
+
+First of all enabling multiarch or 32bit support is not required here it should not be disbaled in the first place.
+
+### AMD
+
+Usually if you run an AMD GPU with open source drivers the required packages should already be installed in case they don't run the following command:
+
+    sudo zypper in kernel-firmware-amdgpu libdrm_amdgpu1 libdrm_amdgpu1-32bit libdrm_radeon1 libdrm_radeon1-32bit
+    
+For Vulkan support for the open source AMD drivers run:
+
+    sudo zypper in libvulkan_radeon libvulkan_radeon-32bit libvulkan1 libvulkan1-32bit vulkan-tools vulkan-validationlayers
```

Comment:
There's no need to separate the 2 steps, they should be combined into one.
Also, average user has no need for vulkan-tools or validation layers.